example/src/App.js

```diff
@@ -150,13 +150,13 @@ export default class SquareImageCropper extends React.Component<
     );
   }
 
-  _crop() {
-    ImageEditor.cropImage(
+  async _crop() {
+    const croppedImageURI = await ImageEditor.cropImage(
       this.state.randomPhoto.uri,
       this._transformData,
-      croppedImageURI => this.setState({croppedImageURI}),
-      cropError => this.setState({cropError}),
-    );
+    ).catch(cropError => this.setState({cropError}));
```

Comment:
Could you use `try catch` instead of chaining catch?

Reply:
Done